crates/hdk/src/lib.rs

```diff
@@ -1,20 +1,41 @@
-//! ## Holochain Development Kit (HDK)
+//! The Holochain Development Kit (HDK) provides high and low level functions for writing Holochain applications.
 //!
-//! The HDK exists to make working with WASM in holochain much easier.
+//! Functions of a Holochain application (hApp) can be organized into reusable components. In Holochain terminology these components are called "zomes".
+//! One or multiple zomes are compiled into a WebAssembly (WASM) binary, referred to as a DNA. All of the DNAs of an application are bundled to a hApp.
 //!
-//! Hopefully you don't notice the WASMness at all and it just feels like Rust 🦀
+//! hApps can be installed on a device that's running a so-called conductor, Holochain's runtime. Clients can then call each zome's functions via Remote Procedure Calls (RPC).
```

Comment:
Is it worth spelling out that these procedure calls could happen on localhost websocket ports served by the conductor or come from other nodes via the network?

Reply:
Yes, included.